src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -29,10 +29,15 @@
  */
 public class AddressBookParser {
 
+    private static final String COMMAND_TYPE_CONTACT = "/c";
+
+    private static final String COMMAND_TYPE_EVENT = "/e";
```

Comment:
I would suggest something like `-c` and `-e` instead of `/c` and `/e` as I believe we should differentiate field specifiers from object-type specifier. Especially since `add /c n/some name` looks kind of strange

Reply:
agreed. i will change these when the above conversation has been resolved.